src/conf/local-file.yaml

```diff
@@ -119,6 +119,7 @@ recommend:
   operation-mode: 1                       # 1: cronjob | 2: one-time-job
   cron-job-time-interval: "1h0m00s"       # format: XhYmZs
   recommend-host-policy: true
+  template-version: "v0.2.2"              # policy template version to be used for recommendation (keep empty to fetches latest)
```

Comment:
Is it possible for me to say `template-version: "latest"` to download/use the latest one?

Reply:
yes @nyrahul we can keep it empty to download the latest.